packages/iota/src/binary/addresses/aliasAddress.ts

```diff
@@ -5,14 +5,14 @@ import { ALIAS_ADDRESS_TYPE, IAliasAddress } from "../../models/addresses/IAlias
 import { SMALL_TYPE_LENGTH } from "../commonDataTypes";
 
 /**
- * The length of an alias address.
+ * The length of an alias id.
  */
-export const ALIAS_ADDRESS_LENGTH: number = 20;
+export const ALIAS_ID_LENGTH: number = 32;
```

Comment:
I'd rather put this constant in  the alias output, and reference it here for the address. The output defines the ID field. Same for NFT ID.

Reply:
That's what i did at first, but it resulted in a very weird issue with serialisation where `MIN_ALIAS_ADDRESS_LENGTH` constant would be `NaN`. Wasted an hour and some sanity trying to fix that and didn't manage too.
I suspect i could be caused by some kind of parsing ordering of the files, not sure.

Reply:
Added a comment for the issue for now.